integration-tests/observability-lgtm/src/test/java/io/quarkus/observability/test/LgtmConfigTestBase.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.observability.test;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+public abstract class LgtmConfigTestBase extends LgtmTestBase {
+
+    @ConfigProperty(name = "grafana.endpoint")
```

Comment:
We had this and I removed it (remember now) because it did not follow the Quarkus namespace convention.
Probably you need to call it `quarkus.observability.grafana.endpoint`.

Reply:
We renamed it so we don't get those warnings, as they are not part of the config interface.

> Probably you need to call it quarkus.observability.grafana.endpoint.

Why?

Reply:
it's the property name conventions for Quarkus. CC @gsmet

Reply:
Yeah, as we figured it out, we had that, but we changed it so we got rid of those pesky warnings ...

Reply:
In any case, we really need to rename this property.

Reply:
@gsmet @radcortez any way to have or add some mechanism, so that properties who don't have config interface 'backup', yet still have `quarkus.` prefix, don't cause warnings ... ?
@geoand ^

Reply:
No, we usually provide a mapping for it (even if not used): https://github.com/quarkusio/quarkus/blob/3f8c3b35fad4b0c56ecd205b828c92ff50a44bd4/core/deployment/src/main/java/io/quarkus/deployment/DebugConfig.java#L8-L17

Reply:
My understanding is that this property isn't exposed to the users? It's just something used in your tests right?

In this case, it's perfectly fine not using the `quarkus.` prefix.

Reply:
You can use `@ConfigDocIgnore`.

Reply:
To be honest, it doesn't make sense. I thought it was being implemented in a different way. After looking to the code again, I see we are not defining a new config property at all, therefore it cannot go into the docs.